.travis.yml

```diff
@@ -10,10 +11,16 @@ services:
 install:
   - pip install molecule docker
 
-script:
+before_script:
   - molecule --version
   - ansible --version
-  - molecule test
+
+env:
```

Comment:
This is a matrix — let's us run scenarios in parallel. We can also add various PostgreSQL versions here and run it all in parallel.

Reply:
nice work man!